src/data/contributors/salmanprottoy.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Md. Salman Hossan Prottoy",
+    "skills": ["Javascript", "C/C++", "PHP", "C#", "Python", "Machine Learning","Data Science"],
+    "cause": "I code for creating oppurtunities for everyone",
```

Comment:
I code for is already the part of the string

Reply:
got it